### ISSUE TITLE
Mailspring account issue

### ISSUE THREAD

**Issue by santhakumar90** [Foundry376/Mailspring#1817]
I have a blank screen after setting up my account for mailspring

**Comment by AlexDunmow** [comment]
#1811

**Comment by santhakumar90** [comment]
ok, When will that issue be solved

**Comment by santhakumar90** [comment]
Please quickly fix the bugs

**Comment by santhakumar90** [comment]
I have used three years mailspring application.no issue but yesterday update new version. did't open the mailspring application. Please let me check the issue ASAP.

**Comment by sisdigital** [comment]
I have a blank screen after setting up my account for mailspring

**Comment by MalteKiefer** [comment]
Yeah I too.
This is the error:

```
➜  Downloads mailspring 
Running database migrations
App load time: 366ms

{"error":null}
Manual update check (updates.getmailspring.com/check/linux/x64/1.7.4-d54ff730/bd503658-e4c7-49ed-9905-2f6c2a27a4f1/stable) returned 204
{ TypeError: Cannot read property 'emailAddress' of undefined
    at _AccountStore.addAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/src/flux/stores/account-store.ts:223:18)
    at OnboardingStore._onFinishAndAddAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/onboarding-store.ts:129:20)
    at EventEmitter.eventHandler (/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:36:22)
    at EventEmitter.emit (/usr/share/mailspring/resources/app.asar/node_modules/eventemitter3/index.js:72:35)
    at Function.trigger (/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:52:26)
    at Object.functor [as finishAndAddAccount] (/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/createAction.js:32:56)
    at onboarding_helpers_1.finalizeAndValidateAccount.then.validated (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/decorators/create-page-for-form.tsx:170:29)
  message: 'Cannot read property \'emailAddress\' of undefined' } { pluginIds: [ 'onboarding' ] }
```
@bengotow

**Comment by timberzzman** [comment]
You can just downgrade the app to 1.7.2 if you want to use it again, and wait for a hotfix.

**Comment by JohanneLun** [comment]
I have something similar. If I start mailspring from the terminal I get this output:
```
Gtk-Message: Failed to load module "canberra-gtk-module"
Gtk-Message: Failed to load module "canberra-gtk-module"
Running database migrations
App load time: 806ms

Running Setup
{"error":null}
```
Then I get the login screen and I seem to be able to login, but after I press login the screen just stays white. Nothing happens, and I have this output:

```
{ TypeError: Cannot read property 'emailAddress' of undefined
    at _AccountStore.addAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/src/flux/stores/account-store.ts:223:18)
    at OnboardingStore._onFinishAndAddAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/onboarding-store.ts:129:20)
    at EventEmitter.eventHandler (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:36:22)
    at EventEmitter.emit (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/eventemitter3/index.js:72:35)
    at Function.trigger (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:52:26)
    at Object.functor [as finishAndAddAccount] (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/createAction.js:32:56)
    at onboarding_helpers_1.finalizeAndValidateAccount.then.validated (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/decorators/create-page-for-form.tsx:170:29)
  message: 'Cannot read property \'emailAddress\' of undefined' } { pluginIds: [ 'onboarding' ] }
Raven: 429 - undefined
```

If I try restarting I get the same output as the first, just without "Running Setup". I then get to the "connect an email account" and try that. I have three different, but I get the same result with all, I just end up with a white screen again and the same as the second output. If I try restarting the same happens. 

I am running Ubuntu 16.04. Before I had no problems, but then I changed the password to one of my accounts and experienced the same as https://github.com/Foundry376/Mailspring/issues/1814 . After struggling with that a while I decided to reinstall to see if that would help, but ended up with all my problems mentioned above. 

I would also like to note that I tried to downgrade to 1.7.2 by using the .deb file from releases but after clicking install in the ubuntu software window, it tells me the version 1.7.4 has been installed instead.

**Comment by timberzzman** [comment]
For doing a downgrade on Debian-like systems, you can uninstall the software and reinstall it from a package with the command: sudo dpkg -i <package_name>. I think ubuntu software take the last version from the repo and install it.

**Comment by asceel** [comment]
I have the same issue :(
![image](https://user-images.githubusercontent.com/58683078/75397984-732b0b80-5909-11ea-9a24-5cd15636dd88.png)
Using snap on Manjaro.

**Comment by edwardzhxi** [comment]
had an error with my 365 account 2 days ago, so I decided to delete and re-add that account... now with the blank screen forever after clicking "connect my account"...

**Comment by amdq71** [comment]
Yo también tengo el mismo problema y comenzó a ocurrir al instalar la última versión, ocurre tanto al instalar el .deb como al instalar el snap.

Cuando habrá una solución???

**Comment by nbdamian** [comment]
Same issue here, Ubuntu 18.04 using Office 365

**Comment by Builderhummel** [comment]
Same here using Manjaro 19.0.1

**Comment by Rostislaved** [comment]
same Ubuntutu 19.04

**Comment by AcherStyx** [comment]
Same on opensuse tumbleweed, I downgrade to 1.7.2 and it work now. But on snapcraft it only provide 1.7.4 :-(

**Comment by JCChaux** [comment]
> Same on opensuse tumbleweed, I downgrade to 1.7.2 and it work now. But on snapcraft it only provide 1.7.4 :-(

Hello,

I have same problem too. Linux Elementary OS and Windows 10.
In Linux I use snap packet but i can't find a way to downgrade. How did you do it with snap packet?

Thanks.

**Comment by AcherStyx** [comment]
> > Same on opensuse tumbleweed, I downgrade to 1.7.2 and it work now. But on snapcraft it only provide 1.7.4 :-(
> 
> Hello,
> 
> I have same problem too. Linux Elementary OS and Windows 10.
> In Linux I use snap packet but i can't find a way to downgrade. How did you do it with snap packet?
> 
> Thanks.
Hi,
I'm sorry but I just downgraded by installing the rpm package. You can find it in the release page.
I would switch back to snap after the bug was fixed.

**Comment by pashtet422** [comment]
same here fedora 31

**Comment by NathanSkene** [comment]
Same here using Ubuntu and trying to add an Office 365 account

**Comment by zeph** [comment]
downgraded to 1.7.2 and it works

**Comment by mtlazaro** [comment]
Same issue here, Ubuntu 16.04 and installation through snap.
I wanted to reconfigure my account settings and blank screen happened after set-up.
I needed to uninstall from snap and install 1.7.2 through .deb package.

Hope this issue can be solved soon.

**Comment by J4RF** [comment]
Same issue on macOS 10.15.3. I also have an existing install on another Mac (also 10.15.3) that gives the same white screen when I go to Preferences > Accounts > Update Connection Settings.

**Comment by ansivirus** [comment]
I had same issue Ubuntu 18.04 mailspring 1.7.4 installed via snap. I removed that version and installed 1.7.5 using snap with edge channel and it works.

**Comment by bengotow** [comment]
Hey folks! Thanks for reporting this and submitting a fix - I'm sorry it's taken me so long to ship a build with the changes. I honestly can't believe it's April already. I'm doing QA tonight and I'll ship the 1.7.5 release with the patch ASAP tomorrow!

**Comment by bengotow** [comment]
Hey folks, thanks for reporting this. Happy to say this has been resolved in 1.7.6 which shipped this afternoon. 🚀  The app should auto-update, but if it doesn't you can also grab the new release from GitHub's Releases tab or from https://getmailspring.com/download. Thanks for your patience and help tracking this down, enjoy the update!
